recipes/r-exactranktests/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set version = '0.8-29' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-exactranktests
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: exactRankTests_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/exactRankTests_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/exactRankTests/exactRankTests_{{ version }}.tar.gz
+  sha256: 21dea6b77a21aae50d3bd657dc7cb4810a5a0f95251e93e0d3592ca5815b446a
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: survival
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('exactRankTests')"  # [not win]
+    - "\"%R%\" -e \"library('exactRankTests')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=exactRankTests
+  license: GPL (>= 2)
```

Comment:
GPL requires that the license be distributed with the package. Please include a `license_file` entry.